deploy/chart/local-path-provisioner/values.yaml

```diff
@@ -74,10 +74,10 @@ podAnnotations: {}
 podSecurityContext: {}
   # runAsNonRoot: true
 
-securityContext: {}
+securityContext:
+  seccompProfile:
+    type: RuntimeDefault
```

Comment:
I'm wondering if we need to enable it by deafult. Or, users can enable it if they need it.

Reply:
I would aim for by default and have folks loosen the security restrictions if they can't run with them.